config_defaults_inc.php

```diff
@@ -1946,6 +1946,13 @@
 $g_ldap_realname_field = 'cn';
 
 /**
+ * Keep a local copy of the MD5 hashed LDAP password (ON) so we can
```

Comment:
Just mention that the password is cached in the database without referencing the hashing type.